File: android/src/main/java/org/axen/flutter/glide/converter/NetworkConverter.java
package org.axen.flutter.glide.converter;

import android.content.Context;

import androidx.annotation.NonNull;

import com.bumptech.glide.RequestBuilder;

public final class NetworkConverter<T> implements ImageConverter<T> {
    @NonNull
    public RequestBuilder<T> convert(@NonNull Context context, @NonNull RequestBuilder<T> builder, @NonNull Object resource) {
        return builder.load((String)resource);
    }
}

File: android/src/main/kotlin/org/axen/flutter/glide/converter/NetworkConverter.kt
package org.axen.flutter.glide.converter

import android.content.Context
import com.bumptech.glide.RequestBuilder
import org.axen.flutter.glide.common.ImageConverter

class NetworkConverter<T> : ImageConverter<T> {
    override fun convert(
        context: Context,
        builder: RequestBuilder<T>,
        resource: Any
    ): RequestBuilder<T> {
        return builder.load(resource as String)
    }
}